app/src/main/java/com/epfl/drawyourpath/database/FirebaseDatabaseUtils.kt

```diff
@@ -373,7 +397,6 @@ object FirebaseDatabaseUtils {
     /**
      * Helper function to transform a milestone object enum with a date into an object to store in the database
      * The tournament id is not take into account since is used as a key for the trophy in the database.
-     * @param tropy to store in the database
```

Comment:
Suggestion the comment of this function is false, maybe you can edit it if you have the time.

Reply:
I can try, let me know if it correct afterwards.